pkg/compute/build.go

```diff
@@ -99,6 +105,8 @@ func (c *BuildCommand) Exec(in io.Reader, out io.Writer) (err error) {
 
 	var toolchain Toolchain
 	switch lang {
+	case "assemblyscript":
+		toolchain = &AssemblyScript{}
```

Comment:
Just to reinforce the earlier point, an empty `struct{}` used only to hang methods that implement some interface is a (very minor!!) yellow flag that the design could be improved.